### DISCUSSION TITLE
Lights not imported with 'import pygfx as gfx'

### DISCUSSION THREAD

**Discussion by SteKme** [pygfx/pygfx#416] [General]
Hi,

please pardon me - I had issues with lights when installing 0.1.9 from pypi 
- examples from Your doc page failed when I tried to run anything with lights - both pyfgx/helpers/_lights and pygfx/objects/_lights seemed not to import, causing gfx.DirectionalLight(), gfx.DirectionalLightHelper() and likes to fail.
Tried latest and pre-latest version available on pip. Windows 10 py37x64

I had to use python.exe -m pip install -U https://github.com/pygfx/pygfx/archive/main.zip and this way, it seems to work fine now.

I just wanted to let You know - indeed, this may be something specific to me... 
Thank You so much for otherwise very cool package!

**Comment by Korijn** [comment]
We have not performed a release since Lights were added. So that explains your issue. Don't worry, a new release is coming soon!
